Marlin/src/module/temperature.h

```diff
@@ -439,6 +442,7 @@ class Temperature {
 
     #if ENABLED(HAS_HOTEND)
       static temp_range_t temp_range[HOTENDS];
+      static constexpr celsius_t hotend_hysteresis = celsius_t((int)TEMP_HYSTERESIS);
```

Comment:
It seems like `celsius_t(UNKNOWN)` should be fine here. If someone cheekily sets the value to `0.5` probably some `#if` statement will blow up first. But if the compiler gets to this point then I think it should be a legal value.

Reply:
0.5 is a legal and accepted value here, since fixp accept both int and float here. It's just that double is not legal in template parameters (there's some template code in Marlin right now with this define).

Reply:
Should be accepted in the latest commit